.github/workflows/ElastixGitHubActions.yml

```diff
@@ -13,25 +13,26 @@ jobs:
     strategy:
       max-parallel: 3
       matrix:
-        os: [ubuntu-18.04, windows-2019, macos-10.15]
+#         os: [windows-2019]
+        os: [macos-10.15, ubuntu-18.04, windows-2019]
         include:
           - os: ubuntu-18.04
             c-compiler: "gcc"
             cxx-compiler: "g++"
-            itk-git-tag: "v5.1.2"
+            itk-git-tag: "0fce3ff86b9242a298a55a42337769eca8e8a4c8"
```

Comment:
Maybe ITK v5.2rc03 ?

Reply:
That build still fails on the CI, Matt advised me to use a more recent one.